Apps/frontend/cypress/e2e/2-control-panel/reset-button.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+
+describe("everything visible on front page", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:5173/");
+  });
+
+  it("displays btn", () => {
+    cy.get(".control-panel").should("be.visible");
```

Comment:
should be `#control-panel`

better would be to give the buttons special classes for cypress like `data-cy="reset-button"`